vscode-plugin/src/client/client.ts

```diff
@@ -70,6 +70,7 @@ export class Client {
     }
 
     private connectionStatus: ConnectionStatus = ConnectionStatus.INIT;
+    private isFirstErrorToLog: boolean = true;
```

Comment:
rename to smth like 'isConnLostLoged', because now it's look like check of Client class